ckanext/push_errors/tests/test_logging.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import logging
+from datetime import datetime
+from unittest.mock import patch
+from ckanext.push_errors.logging import push_message
+from unittest.mock import ANY
+
+logging.basicConfig(level=logging.DEBUG)
+
+
```

Comment:
Better use a class for tests and functions inside the new class 
Something like `class TestPushErrorLogging:`

Reply:
https://github.com/unckan/ckanext-push-errors/pull/9/commits/f35da93cd3d3f68b4ba79f322623bae8b7946814#diff-b69a6054abb7f5561c3f6e63eca099df436a53e321cf89acbed1068fa72cdbd3